### DISCUSSION TITLE
Download from a html url

### DISCUSSION THREAD

**Discussion by Moelf** [fonsp/Pluto.jl#2257] [Pluto development]
Right now we have to download the .jl file and open it, how hard would it be to just give Pluto a `.html` URL and make it work?

**Comment by disberd** [comment]
You can also put the URL of a Pluto notebooks in the `Open File` and it will automatically create a new notebook and put inside the contents of the remote notebook.
Is that what you are after?

**Comment by Moelf** [comment]
That's what I'm talking about but right now you need to put a url pointing to .jl file, not .html (exported by Pluto)

**Comment by fonsp** [comment]
It might be cool to open and edit the HTML file directly, instead of extracting a `.jl` file out of it. In that case, we would auto-save the HTML export instead of just a `.jl` file. cc @ctrekker @dralletje
